### PR TITLE
VLAN to no VLAN connectivity and veth leftovers tests

### PR DESCRIPTION
**Pull request by myakove**
Test that VLAN interface can't ping non VLAN interface.
Test that when VMI is deleted the veth interfaces are deleted as well from the node.

```release-note
NONE
```

### CODE REVIEW

**Comment by myakove** [comment]
ci test please

**Comment by myakove** [comment]
@cynepco3hahue can you please review and merge?

File: vendor/kubevirt.io/kubevirt/tests/utils.go [hunk 1]
```diff
@@ -2274,7 +2274,7 @@ func SkipIfNoSriovDevicePlugin(virtClient kubecli.KubevirtClient) {
 }
 
 func SkipIfNoMultusProvider(virtClient kubecli.KubevirtClient) {
-	_, err := virtClient.ExtensionsV1beta1().DaemonSets(metav1.NamespaceSystem).Get("kube-multus-ds-amd64", metav1.GetOptions{})
+	_, err := virtClient.ExtensionsV1beta1().DaemonSets(metav1.NamespaceSystem).Get("kube-multus-amd64", metav1.GetOptions{})
```
Comment (cynepco3hahue) [inline vendor/kubevirt.io/kubevirt/tests/utils.go]: mm you cannot change files under the vendor directory, next version update will remove it
I think it better to duplicate this method under your code

File: vendor/kubevirt.io/kubevirt/tests/utils.go [hunk 2]
```diff
@@ -28,7 +28,6 @@ import (
 	"fmt"
```
Comment (cynepco3hahue) [inline vendor/kubevirt.io/kubevirt/tests/utils.go]: I see a lot of changes, I think you need to rebase.

**Comment by myakove** [comment]
ci test please

**Pull request merged**